{"version":3,"file":"static/webpack/static/development/pages/_error.js.76771e8668e04de1cc4c.hot-update.js","sources":["webpack:///./node_modules/next/dist/build/webpack/loaders/next-client-pages-loader.js","webpack:///./pages/_error.js"],"sourcesContent":["\n    (window.__NEXT_P=window.__NEXT_P||[]).push([\"/_error\", function() {\n      var mod = require(\"/Users/alee/Documents/Learning/React/react-nextjs-project/pages/_error.js\")\n      if(module.hot) {\n        module.hot.accept(\"/Users/alee/Documents/Learning/React/react-nextjs-project/pages/_error.js\", function() {\n          if(!next.router.components[\"/_error\"]) return\n          var updatedPage = require(\"/Users/alee/Documents/Learning/React/react-nextjs-project/pages/_error.js\")\n          next.router.update(\"/_error\", updatedPage)\n        })\n      }\n      return mod\n    }]);\n  ","import React from 'react';\nimport Link from 'next/link';\nimport Router from 'next/router';\n\nconst errorPage = () => (\n    <div>\n        <h1>Something went wrong.</h1>\n        <p>Try <Link href=\"/\"><a>going back</a></Link></p>\n        <button onClick={() => Router.push('/auth')}>Go to Auth</button>\n    </div>\n);\n\nexport default errorPage;"],"mappings":";;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;AAAA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;ACZA;AACA;AACA;AACA;AACA;AAAA;AACA;AAAA;AAAA;AAAA;AAAA;AAAA;AACA;AAAA;AAAA;AAAA;AAAA;AAAA;AACA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AACA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAJA;AACA;AAOA;;;;;;;;;;;;;;;;A","sourceRoot":""}